pandas/tests/test_frame.py

```diff
@@ -4192,6 +4192,33 @@ def test_astype_cast_nan_int(self):
         df = DataFrame(data={"Values": [1.0, 2.0, 3.0, np.nan]})
         self.assertRaises(ValueError, df.astype, np.int64)
 
+    def test_astype_str(self):
+        # GH9757
+        dts = Series(date_range('2010-01-04', periods=5))
+        tds = Series([Timedelta(x, unit='d') for x in range(5)])
+        ns = Series(range(5))
+        fs = Series([0.0, 0.2, 0.4, 0.6, 0.8])
+
+        df = pd.DataFrame({
```

Comment:
just for consistency can u had an index with a timezone (and also do a test on a time stamp with s timezone separately)